presets/99-default-initrd.preset

```diff
@@ -0,0 +1,10 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+disable *
```

Comment:
maybe add a brief comment here why it makes sense to disable things by default for the initrd